package.json

```diff
@@ -1578,9 +1714,19 @@
           "group": "inline"
         },
         {
```

Comment:
A similar command is missing for `viewPackageSettings`, which would show up in the Project Status View

Reply:
Great find, yes, it was missing. Added now.